### DISCUSSION TITLE
Why Github Education so hard to certification?

### DISCUSSION THREAD

**Discussion by ischaojie** [community/community#70420] [GitHub Education]
### Hi everyone,

Dear github admins, First of all, thank you very much for the educational discount provided by GitHub. I'm a college teacher from China, and I've been planning to use github in my software engineering course to help my students better understand what software engineering is through hands-on process. But I'm going crazy with the github teacher certification process. My questions are as follows: 

1. The school's email system is divided into student's and teacher's email, the school's email is prefixed with stu while the teacher's email is not, but github obviously only recognizes the email with stu prefix, and the teacher's email needs a separate document. My question is, why is there a difference in treatment when it is obvious from the email format that it is a school's domain? 

2. The process of providing teacher qualifications requires a photo to be taken. Since only supporting documents are required, can't this process be replaced with something like uploading documents? Why do you need to take a photo to upload a document? Or support for rear camera shooting? Do you guys know how difficult it is to shoot documents using the front camera?

Thanks :）

**Comment by bakerrya** [comment]
I have the same exact issue. Front facing camera is impossible to take pictures of documents LOL. I don't understand why you can't just upload screenshots.
